tools/nodeset_compiler/backend_open62541_datatypes.py

```diff
@@ -40,7 +46,15 @@ def generateXmlElementCode(value, alloc=False, max_string_length=0):
     return "UA_XMLELEMENT{}({})".format("_ALLOC" if alloc else "", splitStringLiterals(value, max_string_length=max_string_length))
 
 def generateByteStringCode(value, alloc=False, max_string_length=0):
-    return "UA_BYTESTRING{}({})".format("_ALLOC" if alloc else "", splitStringLiterals(value, max_string_length=max_string_length))
+   return "char stringArr?{}! = {};\n\
```

Comment:
Why do you use this cryptic `?` and `!` notation and replace it later? Can't you directly use `[]`? Adding the byte array then has to use the correct brackets.

Reply:
Use 4 spaces indentation

Reply:
The alloc parameter is not used anymore, remove it from the parameters